content/en/docs/concepts/security/controlling-access.md

```diff
@@ -130,6 +130,16 @@ Once a request passes all admission controllers, it is validated using the valid
 for the corresponding API object, and then written to the object store (shown as step **4**).
 
 
+## Auditing
+
+Kubernetes auditing provides a security-relevant, chronological set of records documenting the sequence of actions in a cluster.
+The cluster audits the activities generated by users, by applications that use the Kubernetes API, and by the control plane itself.
```

Comment:
I think these lines provide a good way to introduce the concept. I'd rather stop here and point readers interested in more details to the canonical page for auditing.